src/components/text/text.module.css

```diff
@@ -141,4 +141,10 @@
 /* .labelSemibold, */
 /* .labelBold, */
 /* .quoteItalic, */
-/* .quoteNormal, */
+
+.quoteNormal {
+  font-size: 1.5rem;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 130%;
```

Comment:
This is needed if we want to support multiple lines. If not, we should probably change the Sanity type from `text` to `string`.
```suggestion
  line-height: 130%;
  whitespace: pre-line;
```

**Before**
<img width="540" alt="image" src="https://github.com/user-attachments/assets/552fbe1f-cbf3-404f-bdfd-625141957192">

**After**
<img width="439" alt="image" src="https://github.com/user-attachments/assets/e8643e14-2022-4371-a3a2-dccb4ee4c39e">